packages/files/setup.cfg

```diff
@@ -34,6 +34,7 @@ include_package_data = True
 install_requires =
     galaxy-util
     fs
+    isal
```

Comment:
@mvdbeek Do you remember why you added the `isal` dependency here as well?
I don't think it's currently used?

Reply:
It's used in https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/util/compression_utils.py#L361-L372 or am I missing something ?

Reply:
Ah, why it's used in the files package ? I don't know, it should be enough to just have it be a dependency of data I thuok ?

Reply:
Yep, exactly. I'll drop it in https://github.com/galaxyproject/galaxy/pull/18449 .